eng/common/scripts/common.ps1

```diff
@@ -63,6 +63,7 @@ $GetEmitterNameFn = "Get-${Language}-EmitterName"
 $GetDirectoriesForGenerationFn = "Get-${Language}-DirectoriesForGeneration"
 $UpdateGeneratedSdksFn = "Update-${Language}-GeneratedSdks"
 $IsApiviewStatusCheckRequiredFn = "Get-${Language}-ApiviewStatusCheckRequirement" 
+$DependentPackagesFromPackageSetFn = "Get-${Language}-DependentPackagesFromPackageSet"
```

Comment:
Ok I've re-added trailing whitespace in a rebase so this should actually push out now.

Reply:
Frustration. Still breaks on patch apply.